test/files_test.rb

```diff
@@ -11,18 +11,28 @@ def test_schema_from_file
     refute_valid schema_fixture_path('good_schema_1.json'), { "a" => "bad" }
   end
 
-  def test_data_from_file
+  def test_data_from_file_v3
```

Comment:
Test unit only runs methods starting with `test_` - so this is the correct method name. I've also checked the other test classes and they all seem to have correct test method names